src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_PatternSwitchStatement.cs

```diff
@@ -333,6 +333,7 @@ private BoundExpression TypeTestAndCopyToTemp(BoundExpression input, BoundExpres
                 // invariant: the input has already been tested, to ensure it is not null
                 if (input == temp)
                 {
+                    // this may not be reachable due to https://github.com/dotnet/roslyn/issues/16878
```

Comment:
Yes.

---
In reply to: [100478333](https://github.com/dotnet/roslyn/pull/16928#discussion_r100478333) [](ancestors = 100478333)